pkg/fixtures/writer_ddb.go

```diff
@@ -23,22 +26,35 @@ func DynamoDbFixtureWriterFactory(settings *ddb.Settings) FixtureWriterFactory {
 				WriteCapacityUnits: 1,
 			},
 			Global: settings.Global,
-		})
+		}
+
+		factory := func() ddb.Repository {
```

Comment:
i do not get the reason why we need a factory here

Reply:
solved in private conversation